docs/release/releases/release-1.11.md

```diff
@@ -21,13 +21,13 @@ The following table shows the preliminary dates for the `v1.11` release cycle.
 | Communicate beta to providers                         | Comms Mgr    | Tuesday 15th July 2025     | week 12  |
 | Communicate upcoming code freeze to the community     | Comms Mgr    | Tuesday 15th July 2025     | week 12  |
 | v1.11.0-beta.x released                               | Release Lead | Tuesday 22nd July 2025     | week 13  |
```

Comment:
Can we add another beta here in between. Just so we still have weekly releases from beta.0 until GA

Reply:
@kubernetes-sigs/cluster-api-release-team Is that okay for you?

Reply:
Added onee more beta, PTAL